File: JAVA/Program/Produk.java
public class Produk {
    protected String idProduk, namaProduk;
    protected int harga, stok;

    public Produk(String idProduk, String namaProduk, int harga, int stok) {
        this.idProduk = idProduk;
        this.namaProduk = namaProduk;
        this.harga = harga;
        this.stok = stok;
    }

    public void tampilkanInfo() {
        System.out.println("Produk ID : " + idProduk);
        System.out.println("Nama      : " + namaProduk);
        System.out.println("Harga     : " + harga);
        System.out.println("Stok      : " + stok);
    }
}

File: PYTHON/Program/Produk.py
class Produk:
    def __init__(self, id_produk, nama_produk, harga, stok):
        self.id_produk = id_produk
        self.nama_produk = nama_produk
        self.harga = harga
        self.stok = stok

    def tampilkan_info(self):
        print(f"Produk ID : {self.id_produk}")
        print(f"Nama      : {self.nama_produk}")
        print(f"Harga     : {self.harga}")
        print(f"Stok      : {self.stok}")
